index.html

```diff
@@ -8,37 +8,53 @@
     <link rel="stylesheet" href="styles.css">
     <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/all.css">
     <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+    <link rel="stylesheet" href="mobile.css">
+    <link rel="stylesheet" href="desktop.css">
 </head>
 <body>
-    <header class="toolbar">
-        <a href="#" class="logo">My logo</a>
-        <nav class="fas fa-bars"></nav>
-    </header>
+
+
+        <nav class="nav-bar">
+          <a href="#" class="logo">My logo</a>
+          <i class="fas fa-bars" id="hamburger"></i>
+          
+          <ul class="nav-list">
+            
+            <li class="nav-items">Portfolio</li>
+            <li class="nav-items">About</li>
+            <li class="nav-items">Contact</li>
+          </ul>
+        </nav>
```

Comment:
- [ ] Kindly make sure your design sticks to the Figma design as much as possible. 
Figma design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/193141969-214207b9-7485-4e5f-80ee-90f4aef6e6fe.png)
Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/193142155-30fa7d02-9ee8-400b-ad82-c7c48347a2c9.png)